src/mvAppItem.h

```diff
@@ -314,26 +314,28 @@ GetEntityCommand(mvAppItemType type)
     case mvAppItemType::mvInputFloatMulti:             return "add_input_floatx";
     case mvAppItemType::mvInputDoubleMulti:            return "add_input_doublex";
     case mvAppItemType::mvDragPoint:                   return "add_drag_point";
+    case mvAppItemType::mvDragRect:                    return "add_drag_rect";
     case mvAppItemType::mvDragLine:                    return "add_drag_line";
     case mvAppItemType::mvAnnotation:                  return "add_plot_annotation";
+    case mvAppItemType::mvAxisTag:                     return "add_axis_tag";
     case mvAppItemType::mvLineSeries:                  return "add_line_series";
     case mvAppItemType::mvScatterSeries:               return "add_scatter_series";
     case mvAppItemType::mvStemSeries:                  return "add_stem_series";
     case mvAppItemType::mvStairSeries:                 return "add_stair_series";
     case mvAppItemType::mvBarSeries:                   return "add_bar_series";
+    case mvAppItemType::mvBarGroupSeries:              return "add_bar_group_series";
     case mvAppItemType::mvErrorSeries:                 return "add_error_series";
     case mvAppItemType::mvCustomSeries:                return "add_custom_series";
-    case mvAppItemType::mvVLineSeries:                 return "add_vline_series";
-    case mvAppItemType::mvHLineSeries:                 return "add_hline_series";
+    case mvAppItemType::mvInfLineSeries:               return "add_inf_line_series";
     case mvAppItemType::mvHeatSeries:                  return "add_heat_series";
     case mvAppItemType::mvImageSeries:                 return "add_image_series";
     case mvAppItemType::mvPieSeries:                   return "add_pie_series";
     case mvAppItemType::mvShadeSeries:                 return "add_shade_series";
     case mvAppItemType::mvLabelSeries:                 return "add_text_point";
     case mvAppItemType::mvHistogramSeries:             return "add_histogram_series";
     case mvAppItemType::mv2dHistogramSeries:           return "add_2d_histogram_series";
+    case mvAppItemType::mvDigitalSeries:               return "add_digital_series";
     case mvAppItemType::mvCandleSeries:                return "add_candle_series";
-    case mvAppItemType::mvAreaSeries:                  return "add_area_series";
```

Comment:
`add_area_series`: What's reason for removing this type of series? Looking at the source code I don't see why we need to remove it - since it's not an ImPlot primitive (it's implemented in DPG itself), won't it work fine with the updated ImPlot?

Reply:
Isn't it pretty much useless? And its behaviour isn't the same of `add_shaded_series`?